clarifai/runners/models/model_upload.py

```diff
@@ -212,13 +273,15 @@ def monitor_model_build(self, model_version_id):
 
 def main(folder):
   uploader = ModelUploader(folder)
+  uploader.download_checkpoints()
   uploader.create_dockerfile()
   uploader.upload_model_version()
 
 
 if __name__ == "__main__":
   parser = argparse.ArgumentParser()
-  parser.add_argument('--folder', type=str, help='Folder to tar and upload', required=True)
+  parser.add_argument(
```

Comment:
https://github.com/Clarifai/clarifai-python/tree/master/clarifai/runners/models i wonder if we should add a README.md to that folder to describe how to upload a model? Or should we just add that once you have the examples and we point to that repo in the README?

Reply:
Yeah, I think it would be better once we have examples in the `clarifai/examples`